eng/helix/content/installnode.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+if type -P "node" &>/dev/null; then
+    echo "node is in \$PATH"
+    exit
+fi
+
+node_version=$1
+osname=`uname -s`
+echo $osname
+if [ "$osname" = "Darwin" ]; then
+   platformarch='darwin-x64'
+else
+   platformarch='linux-x64'
+fi
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+output_dir=$DIR/node
```

Comment:
```suggestion
output_dir="$DIR/node"
```
Double quotes for safety in case $DIR contains spaces